Makefile

```diff
@@ -199,7 +199,12 @@ endif
 ## Extension background/serviceworker script.
 BACKGROUND_JS = shared/js/background/background.js
 ifeq ($(type), dev)
-  BACKGROUND_JS := shared/js/background/debug.js $(BACKGROUND_JS)
+  # Developer builds include the devbuilds module for debugging.
+  BACKGROUND_JS += shared/js/background/devbuild.js
+  # Unless reloader=0 is passed, they also contain an auto-reload module.
+  ifneq ($(reloader),0)
+    BACKGROUND_JS += shared/js/background/devbuild-reloader.js
```

Comment:
I'd prefer a solution here that avoids adding more entrypoints as this could be a blocker to switching bundlers. For example, `esbuild` does a 1-1 mapping from entry point files to bundle files. Therefore, this solution (an array of input files), would not be compatible.

Reply:
The background and some of the unit test bundles already have multiple entry points. That is something we will need to resolve anyway, if we switch to a bundler that doesn't support that.

To do that, I think we could have the Makefile generate a intermediate file that imports the modules we need, if there's not another way to do it. I think Make has support for those types of files, and even knows to clear them away again.

Doing that is IMO out of scope for this change, we should worry about it at the point we switch bundler. Happy to help with it when we get there though!